src/internal/operators/distinct.ts

```diff
@@ -57,11 +58,11 @@ import { noop } from '../util/noop';
  * @see {@link distinctUntilKeyChanged}
  *
  * @param {function} [keySelector] Optional function to select which value you want to check as distinct.
- * @param {Observable} [flushes] Optional Observable for flushing the internal HashSet of the operator.
+ * @param {ObservableInput} [flushes] Optional ObservableInput for flushing the internal HashSet of the operator.
```

Comment:
Also, if you can, please remove anything between `{}`. The docs generator scripts are smart enough to understand params' type only by writing their name. If you'd write it like this:

```suggestion
 * @param flushes Optional `ObservableInput` for flushing the internal HashSet of the operator.
```

That would be nice. Fixing it for `keySelector` would be nice as well. Thanks.

Reply:
Thanks, @jakovljevic-mladen. Cheers. Just pushed the changes.